docs/wiki/Whats-new.md

```diff
@@ -51,6 +51,8 @@ This article will be updated as and when changes are made to the above and anyth
 Here's what's changed in Enterprise Scale/Azure Landing Zones:
 
 ### TBD 2023
+- Add Policy initiative [Unused resources driving cost should be avoided (azadvertizer.net)](https://www.azadvertizer.net/azpolicyinitiativesadvertizer/Audit-UnusedResourcesCostOptimization.html)
```

Comment:
```suggestion
- Add new Azure Policy Initiative and assignment (Audit-UnusedResourcesCostOptimization) to audit unused resources that are driving cost at the intermediate root management group
```

Reply:
Good guess on AzAdvertizer, but suggest we not include the link until it's official

Reply:
LOL, color me embarrassed. My bad @JulianHayward!